src/sentry/tagstore/current/models/grouptagkey.py

```diff
@@ -0,0 +1,86 @@
+"""
+sentry.tagstore.current.models.grouptagkey
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2017 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+import six
+
+from django.db import router, transaction, DataError
+
+from sentry.api.serializers import Serializer, register
+from sentry.db.models import (
+    Model, BoundedPositiveIntegerField, BaseManager, sane_repr
+)
+
+
+class GroupTagKey(Model):
+    """
+    Stores a unique tag key name for a group.
+
+    An example key might be "url" or "server_name".
+    """
+    __core__ = False
+
+    project_id = BoundedPositiveIntegerField(db_index=True)
+    group_id = BoundedPositiveIntegerField(db_index=True)
+    environment_id = BoundedPositiveIntegerField()
+    key_id = BoundedPositiveIntegerField()
+    values_seen = BoundedPositiveIntegerField(default=0)
+
+    objects = BaseManager()
+
+    class Meta:
+        app_label = 'sentry'
+        db_table = 'sentry_grouptagkey_current'
+        unique_together = (('project_id', 'group_id', 'key_id'), )
+
+    __repr__ = sane_repr('project_id', 'group_id', 'key_id')
+
+    # TODO: key property to fetch actual key string?
+
+    # TODO: this will have to iterate all of the possible environments a group has?
```

Comment:
I copied this method from legacy because we'll still need to do it. But groups won't have an environment, so I guess this method will have to iterate every possible environment in the project and do this query for each of them so it can get counts correct with respect to envs.